### ISSUE TITLE
`numberOfLines` does not affect previously set text

### ISSUE THREAD

**Issue by phlippieb** [TTTAttributedLabel/TTTAttributedLabel#701]
When working with plain old UILabels, doing something like
```swift
lbl.text = "some long text..."
lbl.numberOfLines = 0
```
will result in the text breaking if needed, even though the number of lines was only set after the text. Doing the same on a TTTAttributedLabel does not result in line breaks.

**Comment by danielchangsoojones** [comment]
Yes, I ran into this problem also. The workaround for now is to place the .numberOfLines before you set .text, but hopefully this gets fixed.
